spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/RewriteDataFilesProcedure.java

```diff
@@ -139,7 +140,10 @@ private RewriteDataFiles checkAndApplyOptions(InternalRow args, RewriteDataFiles
 
   private RewriteDataFiles checkAndApplyStrategy(RewriteDataFiles action, String strategy, SortOrder sortOrder) {
     // caller of this function ensures that between strategy and sortOrder, at least one of them is not null.
-    if (strategy == null || strategy.equalsIgnoreCase("sort")) {
+    if (strategy == null || strategy.equalsIgnoreCase("sort") || strategy.equalsIgnoreCase("zorder")) {
```

Comment:
What I was thinking was we just keep Binpack and sort order as options then we add in a special casing like

If sort_order.startsWith(zorder) {
   zorder
} else {
   Normal sort path
}

And this is what we would do till we have better transform handling for multi-column transforms.

Reply:
there can be some column called `zorder` it will conflict with the assumption?

Reply:
Ok "zorder\s*\(.*\)"